### DISCUSSION TITLE
How to assign colors to various aspects of the snacks Dashboard?

### DISCUSSION THREAD

**Discussion by Jas-SinghFSU** [folke/snacks.nvim#342] [Q&A] [answered]
I can't seem to figure out how I can assign colors to my header, my different section labels, icons, etc. I don't see anything in the documentation either.

There is slight mention of `hl` but that doesn't seem to have any effect. Is it possible to change the dashboard color?

Additionally, what is the command we can assign to a keybind to go to the dashboard?

**Comment by Jas-SinghFSU** [comment]
Sigh... 2 minutes later I find the answer to my first question: https://github.com/folke/snacks.nvim?tab=readme-ov-file#-highlight-groups

Still don't know the command to go to the dashboard however.

**Reply by Jas-SinghFSU** [reply, marked as answer]
```ts
    keys = {
      {
        "<leader><CR>",
        function()
          Snacks.dashboard()
        end,
        desc = "Toggle Dashboard",
      },
    },
```

**Reply by trevorhauter** [reply]
I was searching all over for this today and didn't realize I needed to be overriding highlight groups! Thank you for posting this! For anybody wondering how they might do it, here's an example of how I overrode my dashboard header color
```lua
return {
    "folke/snacks.nvim",
    priority = 1000,
    lazy = false,
    init = function()
        -- load whatever theme you are using
        vim.cmd.colorscheme("gruvbox-material")

        -- Then override snacks.nvim header color like so
        local dashboard_blue = "#89B4FA"
        vim.api.nvim_set_hl(0, "SnacksDashboardHeader", {
            fg = dashboard_blue,
            bold = true,
        })
        ...
    end,
    opts = { dashboard = { enabled = True }}, 
```
